.github/scripts/install_deps.sh

```diff
@@ -13,6 +13,11 @@ os=$(uname -s)
 if [[ $os == Linux ]]; then
   sudo apt-get update
   sudo apt-get install -y attr build-essential cmake curl gettext libattr1-dev ninja-build unzip
+
+  wget https://apt.llvm.org/llvm.sh
```

Comment:
could we add a check before this line to see if the default clang is 17+, then fail the build, so that we will remember to remove this extra install?

Reply:
Hmm, good idea. It should be easy simple enough, I'll take a look.